### PR TITLE
Updated to closure actions in 2 components

### LINKED ISSUE #1519: Use closure actions instead of sendAction in the project.
**Is your feature request related to a problem? Please describe.**

Right now we are using sendAction in various components. We should use closure actions in the project.
Not a huge problem but can be done after handling major issues.

### PR DESCRIPTION
**Pull request by DaKeiser**
Fixes #1519 

#### Short description of what this resolves:
This is an upgrade from using bubble actions to closure actions to avoid deprecations in further versions.

#### Changes proposed in this pull request:

- Updated few component files where sendAction() was used

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by DaKeiser** [comment]
@kushthedude I have updated the code in 2 files. It passed all the checks and if that looks good I can go on to make changes in other files

File: app/components/widgets/forms/time-picker.js [hunk 1]
```diff
@@ -40,9 +40,9 @@ export default Component.extend({
     this.$().calendar(merge(defaultOptions, this.options));
   },
   actions: {
-    onChange() {
+    handleFocusOut() {
       if (this.onChange) {
-        this.sendAction('onChange', this.value);
+        this.onChange(); // invoking method directly.
```
Comment (kushthedude) [inline app/components/widgets/forms/time-picker.js]: Why to rename the onChange ?

Comment (DaKeiser) [reply app/components/widgets/forms/time-picker.js]: yeah I'll keep the same if it's an issue

Comment (kushthedude) [reply app/components/widgets/forms/time-picker.js]: @DaKeiser How is it going ?

Comment (DaKeiser) [reply app/components/widgets/forms/time-picker.js]: Yeah, I am doing it. Facing some issues but yeah I shall submit it by sunday.

**Comment by kushthedude** [comment]
Stale ? 
Please make changes to reopen. Thanks

**Pull request closed without merging**